pkg/resource/snapshot/utility.go

```diff
@@ -0,0 +1,151 @@
+package snapshot
+
+import (
+	"errors"
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
+	ackrequeue "github.com/aws-controllers-k8s/runtime/pkg/requeue"
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	condMsgCurrentlyDeleting string = "snapshot currently being deleted."
+)
+
+var (
+	requeueWaitWhileDeleting = ackrequeue.NeededAfter(
+		errors.New("Delete is in progress."),
```

Comment:
nit: lowercase error messages and remove trailing period
```suggestion
		errors.New("delete is in progress"),
```